x-pack/legacy/plugins/reporting/server/types.d.ts

```diff
@@ -16,7 +16,7 @@ import { ReportingConfig, ReportingConfigType } from './core';
 export interface ReportingSetupDeps {
   elasticsearch: ElasticsearchServiceSetup;
   security: SecurityPluginSetup;
-  usageCollection: UsageCollectionSetup;
+  usageCollection?: UsageCollectionSetup;
```

Comment:
cc @ @afharo - this commit is from your feedback on https://github.com/elastic/kibana/pull/61696#